esp/esp/program/modules/handlers/volunteermanage.py

```diff
@@ -73,7 +75,14 @@ def volunteering(self, request, tl, one, two, module, extra, prog):
             response['Content-Disposition'] = 'attachment; filename=volunteers.csv'
             return response
 
-        if 'op' in request.GET:
+        elif extra == "request_import":
```

Comment:
@benjaminjkraft How would you feel if I just got rid of the confirmation page (since the slots are already saved anyways, see my inline comment), too?